geiger/src/messaging/startListening.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Edge from '../graph/Edge';
+import { EdgeTypes } from '../graph/EdgeTypes';
+import { computeNodeGraphId } from '../graph/label';
+import NodeWithPort from '../graph/NodeWithPort';
+
+/**
+ * Listens to audio context change, and then updates the graph accordingly.
+ */
```

Comment:
This function has an argument `graph`. Use JSDoc annotation to describe the interface. Also if the function does not need JSDoc annotation, use plain comments with `//`.